### ISSUE TITLE
weird addon delay on whitemane

### ISSUE THREAD

**Issue by dididadadududada** [gamefreedomgit/Maelstrom#5815]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
theres a delay with addons that track interrupts in arena here on whitemane. developers act like it isnt happening but many ppl can confirm it bugs out sometimes. u might not be able to fix it but lets stop pretending like it isnt happening

**How to reproduce:**
get any addon that tracks interrupts it doesnt matter which, enter an arena and watch it bug out randomly. it always delays the interrupt with 4 seconds. sometimes it works and sometimes it bugs so u need to test it intensively

**How it should work:**
there shouldn't be a weird ass delay

**Database links:**
literally all addons that tracks interrupts

**Comment by UltraNix** [comment]
`literally all addons that tracks interrupts` Like what?

**Comment by dididadadududada** [comment]
got a video showcasing the issue using omnibar addon which tracks interrupts.
at 0:02 you can see it works fine when the shaman uses wind shear on me, the addon tracks 15 seconds cooldown on wind shear, but then at  0:25 you see the druids skull bash is shown as its on cooldown but he uses it and the addon tracks it with a delay, beginning at 0:28. that is around 4 seconds delay.
you can see the skull bash on my debuffs, it lasts 10 seconds which is the same as the cooldown

https://www.youtube.com/watch?v=W0DziyoPYyQ

**Comment by UltraNix** [comment]
According to this - if Skull Bash is 10 sec cooldown, why addon still shows it's on cooldown (2 sec left) when he actually used against you? It would mean that addon delayed the 1st use of it as well.

**Comment by dididadadududada** [comment]
the whole reason im creating this thread is because addons that track interrupts aren't tracking them correctly, in other words they're sometimes delayed. this is what we're witnessing in the video i posted.
in the video example it works for shaman wind shear but breaks with druids skull bash. like i said its random, sometimes it works and other times it doesn't work. so if u tested it then it means it worked at that given time. if u keep testing it you will see it will sometimes display the incorrect cooldown of interrupts like it did in the video for skull bash. it is entirely random when it happens and it can happen with any interrupt spell like kick, pummel, mind freeze, counterspell etc just like it can happen with any addon that tracks interrupts, for example Juked, Omnibar and Interruptbar etc.

i can make another video showing the same thing if you want but i dont see how that would make any difference as its only happening sometimes. i hope this clears things up abit.

**Comment by SimonBpvp** [comment]
Yeah the combat log is the thing that gets bugged and starts lagging by 2 seconds which throws the addons timing off. I have no idea what causes it but it’s annoying. /reload fixes it temporarily. I wonder if it could be the client we are using. Or if it’s server side? I tried different addons one by one and they all have the problem. If someone tested 2 different clients it would be a big clue

**Comment by Polynomix** [comment]
does it happen outside of arena/bg ?

**Comment by SimonBpvp** [comment]
> does it happen outside of arena/bg ?

I actually think in my experience it only bugs inside of arena but I mostly only play arena anyway. But once it bugs it stays bugged wherever you go.

**Comment by dididadadududada** [comment]
Sorry for the late reply, I was on vacation.

It happens in arenas, but like Simon said it might also happen elsewhere after it happens in arenas. So doing arenas could very well be the trigger. 

It isn't the client since I've tested with different clients, both from clients from whitemane and from other pservers.
